hack/collector-default

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+HOST_PATH=$1
+BUNDLE_DIR=$2
+
+function collect_sysinfo(){
+    cd ${BUNDLE_DIR}
+    mkdir -p hostinfos
+    cd hostinfos
+
+    echo "hostname : " `cat ${HOST_PATH}/etc/hostname` > hostinfo
+    echo "os-release : " `cat ${HOST_PATH}/etc/os-release` >> hostinfo
+    echo "system_info : " `uname -a` >> hostinfo
+
+    # Boot Configuration
+    cat ${HOST_PATH}/boot/config-$(uname -r) > boot_config
+}
+
+function collect_dmesg(){
+    dmesg -HTx &> dmesg.log
+}
+
+function collect_syslog(){
+    tail -c 10m ${HOST_PATH}/var/log/syslog > syslog.log
+}
+
+function collect_kernel_log(){
+    chroot ${HOST_PATH} /usr/bin/journalctl -k > kernel.log
+
+    # also collect logs of previous two boots
+    for ((i=1; i<=2; i++)); do
+    log_name="kernel.log.$i"
+    chroot ${HOST_PATH} /usr/bin/journalctl -k -b $((-i)) > ${log_name}
+    [ ! -s ${log_name} ] && rm -f ${log_name}
+    done
+}
+
+function collect_kubelet_log(){
```

Comment:
BTW, will the support bundle collect each node's logs?

Reply:
I think yes, the `supportbundle-agent` pod will start on each node.
![image](https://user-images.githubusercontent.com/17548901/211155494-775b675b-8621-4823-928a-0fbf75e5f087.png)

Therefore, the kubelet log from each node will be collected.
![image](https://user-images.githubusercontent.com/17548901/211155279-d780fc0c-00cd-47ef-bb5a-b963a7956f94.png)